addons/website/models/website.py

```diff
@@ -480,15 +480,18 @@ def configurator_init(self):
         return r
 
     @api.model
-    def configurator_recommended_themes(self, industry_id, palette):
```

Comment:
Unless I am misunderstanding something:
```
Steps to reproduce:
- Create a new website and go on the theme selection screen.
- Click on the "View more themes" button.

-> The odoo logo becomes very small.
```
... there is no "view more themes" button before this very commit, so this is not a fix... just part of the new implementation. Just remove the steps to reproduce.